android/app/build.gradle

```diff
@@ -151,19 +150,19 @@ android {
 
 dependencies {
 
-    implementation 'com.facebook.soloader:soloader:0.9.0+'
+    implementation 'com.facebook.soloader:soloader:0.10.5'
     implementation "org.jetbrains.kotlin:kotlin-stdlib:${KOTLIN_VERSION}"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${KOTLIN_VERSION}"
 
-    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:1.1.5"
+    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:2.0.4"
 
     implementation fileTree(dir: "libs", include: ["*.jar"])
 
     // The version of react-native is set by the React Native Gradle Plugin
     implementation("com.facebook.react:react-android")
 
     implementation 'androidx.multidex:multidex:2.0.1'
-    implementation "androidx.core:core-splashscreen:1.0.0"
```

Comment:
There is a react-plugging in the package.json that uses it, or does this belongs to the React SDK. If both are in the same, I think then is OK to upgrade.

Reply:
Can you point which SDK needs it?

Reply:
https://github.com/AtB-AS/mittatb-app/blob/ef0ecc6c32b97435062e1ed9060715f02a76c9d8/package.json#L95

Maybe is a dependency of that one, not quite sure, good article.

Reply:
nevermind found it, and it specifies 1.0.0 so I'll revert the change

Reply:
reverted